indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -141,11 +141,20 @@
                             <td class="text-right">{{ item.price }} {{ currency }}</td>
                         </tr>
                     {% endfor %}
+                    {% if registration.registration_form.base_price %}
+                        <tr>
```

Comment:
Not necessary imho. It's billable stuff, so they should follow the style of the rest.